bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.6.3",
+  "version": "1.0.0",
```

Comment:
According to semver, breaking changes can occur before 1.0 at will.

If we release, e.g., 0.7 we have the freedom to decide upfront although since the jQuery usage is pretty basic, i believe that will have no regressions

Another thing that i'd like to propose is putting the dependencies as peerDependencies like we do for Marionette, not sure this will be configured as a breaking change

I think that jQuery 1 and 2 should be supported since there's no reason to not support them